docs/DeveloperGuide.md

```diff
@@ -270,14 +270,32 @@ _{Explain here how the data archiving feature will be implemented}_
+| `* * *`  | Busy Engineering Manager                              | find a job role by name                  | easily locate the job role which are still available                   |
+| `* *`    | Busy Engineering Manager                              | set reminder of interview                | stay organised and track upcoming interview                            |
+| `* *`    | Engineering Manager with sensitive information        | hide private contact details             | protect the privacy of the applicants information in the App           |
+| `* *`    | Engineering Manager with many applicants              | sort the applicants by skill             | prioritise and focus on the most promising candidates                  |
+| `* *`    | Engineering Manager with many applicants              | rank the applicants                      | keep track of the applicants who have performed well                   |
+| `* *`    | Engineering Manager                                   | update an applicant details              | easily update their information on the App                             |
+| `* *`    | Engineering Manager                                   | update a job role                        | easily change the details about the job role                           |
+| `* *`    | Engineering Manager with limited budget               | track the cost per hire                  | ensure that the company budget is not exceeded                         |
+| `* *`    | Team-Oriented Engineering Manager                     | add other interviewer                    | facililate collaboration and delegation of responsibilities            |
+| `*`      | Organised Engineering Manager                         | sort persons by name                     | locate an applicant easily                                             |
+| `*`      | Engineering Manager with many contacts                | import contacts from other file          | add multiple contacts into the App smoothly                            |
+| `*`      | Meticulous Engineering Manager                        | store the applicant's background         | make a more informed choice to benefit the company                     |
+| `*`      | Engineering Manager with multiple rounds of interview | track the progress of each candidate     | maintain a clear overview of our recruitment efforts                   |
+| `*`      | New Engineering Manager                               | analyse the performance of the interview | make improvements to my interview processes                            |
+| `*`      | Helpful Engineering Manager                           | provide feedback to the applicant        | offer constructive advice to help them improve their skills            |
+| `*`      | Long user of the app                                  | provide feedback to the developer        | suggest improvements and enhancements for the app                      |
+
```

Comment:
Perhaps could add:
- As an Engineering manager adding an interview, I want to input the interview start time in different formats so that I can <insert benefit of having multiple formats>.